gallery/src/styles/NxCard/NxCardVerticalAlternateExample.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { faShapes, faChartLine } from '@fortawesome/free-solid-svg-icons';
+
+import { NxFontAwesomeIcon } from '@sonatype/react-shared-components';
+
+export default function NxCardVerticalAltExample() {
```

Comment:
Lets rename these to the consistent terminology

Reply:
> Lets rename these to the consistent terminology

https://github.com/sonatype/sonatype-react-shared-components/pull/170/commits/38ec97d38d3854439f9f6e4e7fd34936d4cd01f6